File: TpPoo_cpp/tp2_ex1/Personne.cpp
//
// Created by amani on 01/03/24.
//

#include "Personne.h"
Personne::Personne(int age , string nom) {
    this->nom=nom;
    this->age=age;
}
 int Personne::get_age(){
    return age;
}
string Personne:: get_nom(){
    return nom;
}
void Personne::set_age(int age){
    this->age=age;
}
void Personne:: set_nom(string nom){
    this->nom=nom;
}
void Personne::affiche() {
    cout<<"le nom de la personne est : "<<nom<<endl;
    cout<<"l'age de la personne est : "<<age<<endl;
}

File: Poo_java/tp2/tp2_ex1/src/Personne.java
public class Personne {
        private int age;
        private String nom;

        public Personne(int age, String nom) {
            this.nom = nom;
            this.age = age;
        }

        public int getAge() {
            return age;
        }

        public String getNom() {
            return nom;
        }

        public void setAge(int age) {
            this.age = age;
        }

        public void setNom(String nom) {
            this.nom = nom;
        }

        public void affiche() {
            System.out.println("le nom de la personne est : " + nom);
            System.out.println("l'age de la personne est : " + age);
        }
    }

